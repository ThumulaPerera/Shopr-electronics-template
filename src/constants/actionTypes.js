export const AUTH_ACTION_TYPES = {
    LOGIN_SUCCESS : 'LOGIN_SUCCESS',
    LOGIN_ERROR : 'LOGIN_ERROR',
    SIGNOUT_SUCCESS : 'SIGNOUT_SUCCESS',
    SIGNOUT_ERROR : 'SIGNOUT_ERROR',
}

export const CART_ACTION_TYPES = {
    DELETE_IN_PROGRESS : 'DELETE_IN_PROGRESS',
    DELETE_SUCCESS : 'DELETE_SUCCESS',
    DELETE_ERROR : 'DELETE_ERROR',
    EDIT_IN_PROGRESS : 'EDIT_IN_PROGRESS',
    EDIT_SUCCESS : 'EDIT_SUCCESS',
    EDIT_ERROR : 'EDIT_ERROR',
}

