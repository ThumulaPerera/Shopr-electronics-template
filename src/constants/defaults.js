/* eslint-disable import/prefer-default-export */
export const defaultImgUrl = 'https://firebasestorage.googleapis.com/v0/b/ecom-cse.appspot.com/o/electronics-template%2Fdefault%20images%2FdefaultItem.jpg?alt=media&token=25d58086-a46e-42dc-9228-82a3b1a116d9';

export const browseProductsPageImgUrl = 'https://firebasestorage.googleapis.com/v0/b/ecom-cse.appspot.com/o/electronics-template%2Ftemplate%20screenshots%2Fbrowse%20products%20page.jpg?alt=media&token=1b13816e-ff59-4482-8301-e30d52a89162';
export const itemPageImgUrl = 'https://firebasestorage.googleapis.com/v0/b/ecom-cse.appspot.com/o/electronics-template%2Ftemplate%20screenshots%2Fitem%20page.jpg?alt=media&token=54839d86-7da3-419d-a5a8-19343289051b';
export const cartPageImgUrl = 'https://firebasestorage.googleapis.com/v0/b/ecom-cse.appspot.com/o/electronics-template%2Ftemplate%20screenshots%2Fcart%20page.jpg?alt=media&token=7cd2aed4-52a6-42a1-a154-a078e3f4861b';
export const myPurshasesPageImgUrl = 'https://firebasestorage.googleapis.com/v0/b/ecom-cse.appspot.com/o/electronics-template%2Ftemplate%20screenshots%2Forders%20page.jpg?alt=media&token=7293f018-0fbd-4eb7-8c47-af0ded528358';
