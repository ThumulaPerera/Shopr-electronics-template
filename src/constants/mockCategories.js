/* eslint-disable import/prefer-default-export */
export const categories = {
  1: {
    name: 'Computers and Laptops',
  },
  2: {
    name: 'Cameras and Accessories',
  },
  3: {
    name: 'Smart Phones and Tablets',
  },
  4: {
    name: 'TVs and Home Entertainmnt',
  },
  5: {
    name: 'Batteries and Power',
  },
};
