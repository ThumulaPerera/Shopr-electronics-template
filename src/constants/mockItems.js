export const items = {
    1: {
        name: "camera 1",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 1",
        currency: "$",
        price: "100",
        rating: 1
    },
    2: {
        name: "camera 2",
        imageURL: "https://www.sony-asia.com/image/f6fa6732135c962d2a79f89a9ff3ee6a?fmt=pjpeg&wid=1014&hei=396&bgcolor=F1F5F9&bgc=F1F5F9",
        description: "description of camera 2",
        currency: "$",
        price: "200",
        rating: 2
    },
    3: {
        name: "camera 3",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 3",
        currency: "$",
        price: "300",
        rating: 3
    },
    4: {
        name: "camera 4",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 4",
        currency: "$",
        price: "400",
        rating: 4
    },
    5: {
        name: "camera 1",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 1",
        currency: "$",
        price: "100",
        rating: 1
    },
    6: {
        name: "camera 2",
        imageURL: "https://www.sony-asia.com/image/f6fa6732135c962d2a79f89a9ff3ee6a?fmt=pjpeg&wid=1014&hei=396&bgcolor=F1F5F9&bgc=F1F5F9",
        description: "description of camera 2",
        currency: "$",
        price: "200",
        rating: 2
    },
    7: {
        name: "camera 3",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 3",
        currency: "$",
        price: "300",
        rating: 3
    },
    8: {
        name: "camera 4",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 4",
        currency: "$",
        price: "400",
        rating: 4
    },
    9: {
        name: "camera 3",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 3",
        currency: "$",
        price: "300",
        rating: 3
    },
    10: {
        name: "camera 4",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 4",
        currency: "$",
        price: "400",
        rating: 4
    },
    11: {
        name: "camera 1",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 1",
        currency: "$",
        price: "100",
        rating: 1
    },
    12: {
        name: "camera 2",
        imageURL: "https://www.sony-asia.com/image/f6fa6732135c962d2a79f89a9ff3ee6a?fmt=pjpeg&wid=1014&hei=396&bgcolor=F1F5F9&bgc=F1F5F9",
        description: "description of camera 2",
        currency: "$",
        price: "200",
        rating: 2
    },
    13: {
        name: "camera 3",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 3",
        currency: "$",
        price: "300",
        rating: 3
    },
    14: {
        name: "camera 4",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 4",
        currency: "$",
        price: "400",
        rating: 4
    },
    15: {
        name: "camera 1",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 1",
        currency: "$",
        price: "100",
        rating: 1
    },
    16: {
        name: "camera 2",
        imageURL: "https://www.sony-asia.com/image/f6fa6732135c962d2a79f89a9ff3ee6a?fmt=pjpeg&wid=1014&hei=396&bgcolor=F1F5F9&bgc=F1F5F9",
        description: "description of camera 2",
        currency: "$",
        price: "200",
        rating: 2
    },
    17: {
        name: "camera 3",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 3",
        currency: "$",
        price: "300",
        rating: 3
    },
    18: {
        name: "camera 4",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 4",
        currency: "$",
        price: "400",
        rating: 4
    },
    19: {
        name: "camera 3",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 3",
        currency: "$",
        price: "300",
        rating: 3
    },
    20: {
        name: "camera 4",
        imageURL: "https://d2yisiyyppbfr4.cloudfront.net/catalog/product/cache/image/e9c3970ab036de70892d86c6d221abfe/1/5/1519617660000_img_949161_1.jpg",
        description: "description of camera 4",
        currency: "$",
        price: "400",
        rating: 4
    },
}