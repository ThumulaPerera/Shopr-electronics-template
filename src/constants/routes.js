export const MAIN_ROUTE = '/:storeID';

export const HOME_ROUTE = '';
export const MY_PURCHASES_ROUTE = '/MyPurchases';
export const CART_ROUTE = '/Cart';
export const PROFILE_ROUTE = '/Profile';

