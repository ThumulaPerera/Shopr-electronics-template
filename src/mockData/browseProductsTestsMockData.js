/* eslint-disable import/prefer-default-export */

export const minItems = [
  {
    id: '5ed1112909f2ff2d85b2c21b',
    basePrice: 3835.16,
    category: 3,
    deleted: true,
    description: 'Sint elit dolore pariatur in occaecat pariatur amet est officia elit ea enim. Enim amet enim elit ut nulla sit in minim eu ea. Consequat duis anim laboris laborum officia exercitation amet reprehenderit deserunt consequat fugiat ea aute ex. Ex deserunt labore in sunt ex proident sit ad. Ad elit excepteur irure eiusmod nisi Lorem eu qui ipsum ea. Ipsum excepteur non et velit occaecat duis ut.\r\n',
    name: 'fugiat',
    visible: true,
  },
  {
    id: '5ed111295a728d32a9ca49b7',
    basePrice: 1684.05,
    category: 2,
    deleted: false,
    description: 'Velit cillum veniam proident in deserunt voluptate cillum. Exercitation velit cillum id commodo aliquip dolor reprehenderit dolore consectetur. Non dolor do cillum nisi. Exercitation sint consectetur irure culpa laboris Lorem reprehenderit sit occaecat occaecat do aute esse. Irure minim nostrud sint adipisicing ipsum eu aliquip.\r\n',
    name: 'dolor',
    visible: true,
  },
  {
    id: '5ed1112908cf459fc5101684',
    basePrice: 1921.53,
    category: 0,
    deleted: true,
    description: 'Quis commodo occaecat ut in ad quis. Anim eiusmod dolore Lorem sunt eu sint adipisicing officia. Officia laboris ad dolor esse aliqua.\r\n',
    name: 'laborum',
    visible: false,
  },
  {
    id: '5ed111296d4090b28f206d38',
    basePrice: 3172.05,
    category: 2,
    deleted: false,
    description: 'Id fugiat consectetur proident nisi sunt quis sint duis minim adipisicing duis. Do enim amet do sit adipisicing elit qui consectetur. Elit laborum ullamco amet eu magna pariatur labore ad ea adipisicing ullamco. Cillum non aliqua aliqua ullamco voluptate voluptate ipsum nulla officia sunt. Excepteur velit aliquip ipsum pariatur laborum qui aliqua sunt. Ullamco pariatur consectetur amet dolor sit.\r\n',
    name: 'ipsum',
    visible: true,
  },
  {
    id: '5ed11129a3c7710259fa19e1',
    basePrice: 4388.14,
    category: 2,
    deleted: false,
    description: 'Est cupidatat est adipisicing est enim magna sit duis ad culpa Lorem labore. Nulla commodo laboris ut excepteur labore do ipsum sit proident officia. Elit esse non in minim in. Laboris laboris dolore tempor mollit consequat culpa id amet. Aliquip commodo cupidatat duis ad ut ex cupidatat in et sunt officia.\r\n',
    name: 'officia',
    visible: false,
  },
  {
    id: '5ed11129fb522bfdb34ebe21',
    basePrice: 3317.43,
    category: 3,
    deleted: false,
    description: 'Lorem veniam tempor est fugiat proident sit irure nisi veniam nostrud. Dolor id fugiat laboris elit cupidatat aute ad labore eu fugiat. Enim voluptate irure id minim amet anim occaecat officia sint pariatur laboris velit non consectetur. Pariatur est mollit sunt id laboris mollit adipisicing voluptate culpa eiusmod magna nisi. Pariatur dolore labore magna enim aute qui velit officia adipisicing sit ad. Laboris dolor nisi adipisicing veniam est veniam duis labore id ipsum dolor eiusmod. Est amet ex ex adipisicing in amet sit anim magna aliqua.\r\n',
    name: 'mollit',
    visible: false,
  },
  {
    id: '5ed11129f38a4da11e536dec',
    basePrice: 3623.22,
    category: 3,
    deleted: false,
    description: 'Cillum nulla nulla commodo sint laborum aliquip irure proident mollit dolore nostrud laboris irure. In mollit tempor fugiat aute nisi pariatur labore nostrud eiusmod duis. Laborum aute laborum do sit duis est excepteur sit Lorem eu dolor pariatur. In ea deserunt amet do adipisicing. Exercitation officia irure labore commodo voluptate adipisicing sit nisi consectetur occaecat non magna nisi. Fugiat qui ipsum nulla nisi eu et dolor tempor proident cupidatat proident ipsum cillum culpa.\r\n',
    name: 'mollit',
    visible: false,
  },
  {
    id: '5ed11129bf7508bf980aaebc',
    basePrice: 3488.74,
    category: 2,
    deleted: true,
    description: 'Sit exercitation ullamco anim mollit dolor nulla ut nisi occaecat in. Occaecat ut qui culpa qui laboris sunt. Consequat proident et eiusmod magna. Ullamco occaecat voluptate qui id nulla minim non velit commodo. Nostrud enim sunt sint magna ullamco officia reprehenderit non cupidatat minim elit. Aliqua adipisicing ex nostrud aute eu do ullamco. Excepteur pariatur nisi qui occaecat nisi veniam in.\r\n',
    name: 'eiusmod',
    visible: true,
  },
  {
    id: '5ed11129636830919c3d2175',
    basePrice: 2480.81,
    category: 0,
    deleted: false,
    description: 'Irure veniam magna labore nisi aliquip reprehenderit aliquip. Reprehenderit ex eu nisi esse id commodo nisi exercitation sunt dolor ad officia consectetur. Dolor laboris occaecat Lorem et id cillum dolor occaecat nisi.\r\n',
    name: 'nulla',
    visible: false,
  },
  {
    id: '5ed111295cc737557bf875ce',
    basePrice: 2468.8,
    category: 3,
    deleted: false,
    description: 'Irure eiusmod officia cupidatat veniam. Exercitation anim laborum cupidatat tempor irure do. Ut ad sunt id nostrud eiusmod mollit mollit. Officia sint eu id exercitation nisi consequat id dolore.\r\n',
    name: 'excepteur',
    visible: true,
  },
];

export const sellerStore = {
  categories: [
    {
      name: 'category 0',
    },
    {
      name: 'category 1',
    },
    {
      name: 'category 2',
    },
    {
      name: 'category 3',
    },
    {
      name: 'category 4',
    },
  ],
  currency: 'USD',
  email: 'hendersonreed@orbiflex.com',
  enableChatbot: true,
  enableInventoryManagement: false,
  enableRating: true,
  verified: false,
  storeName: 'ut qui',
  storeCustomization: {},
};
