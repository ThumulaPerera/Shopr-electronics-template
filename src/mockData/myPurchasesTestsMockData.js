/* eslint-disable import/prefer-default-export */
export const items = {
  '5ed111295a728d32a9ca49b7': {
    id: '5ed111295a728d32a9ca49b7',
    basePrice: 100,
    category: 2,
    deleted: false,
    description: 'Velit cillum veniam proident in deserunt voluptate cillum. Exercitation velit cillum id commodo aliquip dolor reprehenderit dolore consectetur. Non dolor do cillum nisi. Exercitation sint consectetur irure culpa laboris Lorem reprehenderit sit occaecat occaecat do aute esse. Irure minim nostrud sint adipisicing ipsum eu aliquip.\r\n',
    name: 'dolor',
    visible: true,
    variants: [
      {
        title: 'variant title 1',
        attributes: [
          {
            attribute: 'variant attribute 11',
          },
          {
            attribute: 'variant attribute 12',
          },
        ],
      },
      {
        title: 'variant title 2',
        attributes: [
          {
            attribute: 'variant attribute 21',
          },
          {
            attribute: 'variant attribute 22',
          },
        ],
      },
    ],
    subItems: [
      {
        price: 100,
        stock: 10,
        variants: [
          'variant attribute 11',
          'variant attribute 21',
        ],
      },
      {
        price: 100,
        stock: 10,
        variants: [
          'variant attribute 12',
          'variant attribute 21',
        ],
      },
      {
        price: 100,
        stock: 10,
        variants: [
          'variant attribute 11',
          'variant attribute 22',
        ],
      },
      {
        price: 100,
        stock: 10,
        variants: [
          'variant attribute 12',
          'variant attribute 22',
        ],
      },
    ],
  },
  '5ed111296d4090b28f206d38': {
    id: '5ed111296d4090b28f206d38',
    basePrice: 3172.05,
    category: 2,
    deleted: false,
    description: 'Id fugiat consectetur proident nisi sunt quis sint duis minim adipisicing duis. Do enim amet do sit adipisicing elit qui consectetur. Elit laborum ullamco amet eu magna pariatur labore ad ea adipisicing ullamco. Cillum non aliqua aliqua ullamco voluptate voluptate ipsum nulla officia sunt. Excepteur velit aliquip ipsum pariatur laborum qui aliqua sunt. Ullamco pariatur consectetur amet dolor sit.\r\n',
    name: 'ipsum',
    visible: true,
  },
  '5ed111295cc737557bf875ce': {
    id: '5ed111295cc737557bf875ce',
    basePrice: 2468.8,
    category: 3,
    deleted: false,
    description: 'Irure eiusmod officia cupidatat veniam. Exercitation anim laborum cupidatat tempor irure do. Ut ad sunt id nostrud eiusmod mollit mollit. Officia sint eu id exercitation nisi consequat id dolore.\r\n',
    name: 'excepteur',
    visible: true,
  },
};

export const sellerStore = {
  categories: [
    {
      name: 'category 0',
    },
    {
      name: 'category 1',
    },
    {
      name: 'category 2',
    },
    {
      name: 'category 3',
    },
    {
      name: 'category 4',
    },
  ],
  currency: 'USD',
  email: 'hendersonreed@orbiflex.com',
  enableChatbot: true,
  enableInventoryManagement: true,
  enableRating: true,
  verified: true,
  storeName: 'ut qui',
  storeCustomization: {},
  merchantId: '123456',
};

export const orders = [
  {
    id: 'order123',
    orderStates: [
      {
        date: {
          seconds: 1590234479,
          nanoseconds: 947000000,
        },
        stateId: 0,
      },
    ],
    buyer: 'buyer123',
    paymentMethod: 'paypal',
    totalPrice: 3200,
    shippingAddress: {},
    orderItems: [

      {
        item: '5ed111295a728d32a9ca49b7',
        noOfItems: 1,
        subItem: 0,
        unitPrice: 100,
      },

    ],
    date: {
      seconds: 1590234479,
      nanoseconds: 947000000,
    },
  },
];
