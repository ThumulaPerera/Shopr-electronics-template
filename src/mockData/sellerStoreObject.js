/* eslint-disable import/prefer-default-export */
export const sellerStore = {
  categories: [
    {
      name: 'category 0',
    },
    {
      name: 'category 1',
    },
    {
      name: 'category 2',
    },
    {
      name: 'category 3',
    },
    {
      name: 'category 4',
    },
  ],
  currency: 'USD',
  email: 'hendersonreed@orbiflex.com',
  enableChatbot: true,
  enableInventoryManagement: false,
  enableRating: true,
  verified: false,
  storeName: 'ut qui',
  storeCustomization: {},
};
