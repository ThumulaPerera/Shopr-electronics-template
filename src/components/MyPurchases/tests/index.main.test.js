/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';

import { Route, MemoryRouter } from 'react-router-dom';
import { MyPurchases as UnconnectedMyPurchases } from '../index';

import { items, orders, sellerStore } from '../../../mockData/myPurchasesTestsMockData';
import { MAIN_ROUTE, MY_PURCHASES_ROUTE } from '../../../constants/routes';


jest.unmock('react-redux-firebase');
const rrf = require.requireActual('react-redux-firebase');

let component;

beforeEach(() => {
  const addReview = jest.fn();
  const toDate = jest.fn(() => new Date(2018, 11, 24, 10, 33, 30));

  orders.forEach((order) => {
    // eslint-disable-next-line no-param-reassign
    order.date.toDate = toDate;
    return null;
  });

  rrf.isLoaded = jest.fn(() => true);
  component = renderer.create(
    <MemoryRouter initialEntries={[{ pathname: `/storeId123${MY_PURCHASES_ROUTE}`, key: 'testKey' }]}>
      <Route path={`${MAIN_ROUTE}${MY_PURCHASES_ROUTE}`}>
        <UnconnectedMyPurchases
          auth={{ uid: '123' }}
          items={items}
          currency={sellerStore.currency}
          color={null}
          ratingEnabled={sellerStore.enableRating}
          orders={orders}
          orderStates={
              [
                'Paid',
                'Shipped',
                'Delivered',
              ]
          }
          addReview={addReview}
          changeInProgress={false}
          match={{
            params: {
              storeID: 'store123',
            },
          }}
          storeName={sellerStore.storeName}
        />
        ,
      </Route>
    </MemoryRouter>
    ,
  );
});

afterEach(cleanup);

it('renders with given props', () => {
  expect(component.toJSON()).toMatchSnapshot();
});
