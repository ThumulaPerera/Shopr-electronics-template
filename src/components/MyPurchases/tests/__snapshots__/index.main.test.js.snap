// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with given props 1`] = `
Array [
  <div
    className="ui basic segment"
  >
    <div>
      <div
        className="ui segment"
      >
        <div
          className="ui items"
        >
          <div
            className="item"
          >
            <div
              className="content"
            >
              <h4>
                Order ID :
                 
                order123
              </h4>
              <div
                className="ui divider"
              />
              <div
                className="meta"
              >
                <h4>
                  Order Items
                </h4>
              </div>
              <div
                className="accordion ui"
              >
                <div>
                  <div
                    className="title"
                    onClick={[Function]}
                  >
                    <i
                      aria-hidden="true"
                      className="dropdown icon"
                      onClick={[Function]}
                    />
                    dolor
                  </div>
                  <div
                    className="content"
                  >
                    <div
                      className="ui items"
                    >
                      <div
                        className="item"
                      >
                        <div
                          className="image"
                        >
                          <img
                            src="https://firebasestorage.googleapis.com/v0/b/ecom-cse.appspot.com/o/electronics-template%2Fdefault%20images%2FdefaultItem.jpg?alt=media&token=25d58086-a46e-42dc-9228-82a3b1a116d9"
                          />
                        </div>
                        <div
                          className="content"
                        >
                          <a
                            className="header"
                            href="/store123/Items/5ed111295a728d32a9ca49b7"
                          >
                            dolor
                          </a>
                          <div
                            className="ui two column grid"
                          >
                            <div
                              className="column"
                            >
                              <div
                                className="accordion ui"
                              >
                                <div>
                                  <div
                                    className="title"
                                    onClick={[Function]}
                                  >
                                    <i
                                      aria-hidden="true"
                                      className="dropdown icon"
                                      onClick={[Function]}
                                    />
                                    Variants
                                  </div>
                                  <div
                                    className="content"
                                  >
                                    <div>
                                      <b>
                                        variant title 1
                                         
                                        :
                                      </b>
                                      <p>
                                        variant attribute 11
                                      </p>
                                      <br />
                                    </div>
                                    <div>
                                      <b>
                                        variant title 2
                                         
                                        :
                                      </b>
                                      <p>
                                        variant attribute 21
                                      </p>
                                      <br />
                                    </div>
                                  </div>
                                </div>
                              </div>
                            </div>
                            <div
                              className="column"
                            >
                              <div
                                className="meta"
                              >
                                Unit price :
                              </div>
                              <div
                                className="description"
                              >
                                USD 100.00
                              </div>
                              <div
                                className="meta"
                              >
                                Quantity :
                              </div>
                              <div
                                className="description"
                              >
                                x
                                 
                                1
                              </div>
                              <div
                                className="meta"
                              >
                                <b>
                                  Sub Total :
                                </b>
                              </div>
                              <div
                                className="description"
                              >
                                <h4>
                                  <b>
                                    USD 100.00
                                  </b>
                                </h4>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="ui divider"
              />
              <div
                className="ui two column grid"
              >
                <div
                  className="column"
                >
                  <div
                    className="meta"
                  >
                    Order Date :
                  </div>
                  <div
                    className="description"
                  >
                    Mon Dec 24 2018 10:33:30 GMT+0530 (India Standard Time)
                  </div>
                  <div
                    className="meta"
                  >
                    Order State :
                  </div>
                  <div
                    className="description"
                  >
                    Paid
                  </div>
                  <div
                    className="meta"
                  >
                    Shipping Address :
                  </div>
                  <div
                    className="description"
                  >
                    
                  </div>
                </div>
                <div
                  className="column"
                >
                  <div
                    className="meta"
                  >
                    Total Price:
                  </div>
                  <div
                    className="description"
                  >
                    <h1>
                      USD 3200.00
                    </h1>
                  </div>
                  <div
                    className="meta"
                  >
                    Payment Method :
                  </div>
                  <div
                    className="description"
                  >
                    paypal
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ui hidden divider"
      />
    </div>
  </div>,
  ",",
]
`;
